docroot/themes/contrib/civictheme/includes/cards.inc

```diff
@@ -140,4 +140,5 @@ function _civictheme_preprocess_node__civictheme_event__summary(&$variables) {
     $value = $node->get('field_c_n_body')->getValue();
     $variables['summary'] = $value[0]['value'];
   }
+  _civictheme_preprocess_node__date($variables);
```

Comment:
@joshua-salsadigital 
looks like we already have a call to `_civictheme_preprocess_node__date()` https://github.com/salsadigitalauorg/civictheme_source/pull/501/files#diff-b259f27509fa1f724a28abf4bde84d0ebf29f95c0668a4576d582460a50941b7R30

But I dunno why it does not apply.

in any way - can you please not put Date field-related processor in a Summary field